pkg/notes/document/template.go

```diff
@@ -63,6 +63,24 @@ filename | sha512 hash
 {{- end -}}
 # Changelog since {{$PreviousRevision}}
 
+{{with .CVEList -}}
+## Important Security Information
+
+This release contains changes that address the following vulnerabilities:
+{{range .}}
+### {{.Title}}
```

Comment:
I would like the CVE number to be included in the title, e.g. `CVE-2020-8559: Privilege escalation from compromised node to cluster`

I'm not sure if we just want to make it convention to set the title like that, or explicitly break the CVE into a separate field.

Reply:
Done :+1: